### PR TITLE
Explain sse

### PR DESCRIPTION
**Pull request by waldemarnt**

### CODE REVIEW

**Comment by glauco** [comment]
@bltavares It's happening!

File: source/includes/telemetry_api/_sse.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+By appending a `/changes/stream` to the endpoints, a connection will start to
+send information whenever the data changes in the background, reducing the
+latency of the connection and supporting live data for streaming.
+
+> Example of using the `/changes/stream` endpoint
+
+```shell
+curl -X GET \
+    --header "Authorization: Bearer {YOUR ACCESS TOKEN}" \
```
Comment (glauco) [inline source/includes/telemetry_api/_sse.md]: s/{YOUR ACCESS TOKEN}/YOUR ACCESS TOKEN/

**Comment by jeffhsta** [comment]
:+1:

**Comment by lucaslago** [comment]
👍

**Comment by carolinaborim** [comment]
👍

**Comment by glauco** [comment]
🐑 🇮🇹

**Pull request merged**